### ISSUE TITLE
How to add fonts?

### ISSUE THREAD

**Issue by xKennyHDx** [rpgboss/rpgboss#498]
Im working on a game and i want to use a different font

**Comment by FaelLiberatori** [comment]
Click on "Database" on the top menu bar and then in the "System" tab.

Inside this windows we have 3 different options that you may want to know about:

- `Message_font`: use it to change the game font;
- `Font Size`: the size you want the font to be;
- `String Input Characters`: you have to put here all characters you want to use in the game. You only will edit it if you want to use an uncommon character, for example, to translate the game to chines or to use the "@" character in the game.

**Comment by xKennyHDx** [comment]
hmm.. Is it possible to have change the font mid game?

**Comment by FaelLiberatori** [comment]
Nope, you will have to implement it by yourself if you really want it.
If you know the basic of Scala it should be ok, all you have to do is create a font manager class to manage the fonts and integrate it with the message box.
Check out this file:
 `https://github.com/rpgboss/rpgboss/blob/master/core/src/main/scala/rpgboss/model/resource/MsgFont.scala`
It's the font class, you have to create a manager for them.
Also, take a look at this one:
`https://github.com/rpgboss/rpgboss/blob/master/desktop/src/main/scala/rpgboss/editor/dialog/db/MessagesPanel.scala`
It's the message box one. You will have to choose which font to use in each situation inside these panels.

Good luck. If you do this, please let us know.
